btcwallet.go

```diff
@@ -5,12 +5,12 @@
 package main
```

Comment:
package should be renamed such as it will be used by other apps.

Reply:
the idea was to do as little changes to the wallet's code as possible. the main package won't be imported anyway, but other packages
does this solve your concern?